server.js

```diff
@@ -0,0 +1,26 @@
+var fs = require('fs');
+var express = require('express');
+var bodyParser = require('body-parser');
+var app = express();
+var stringifyFile;
```

Comment:
try don't use global variables when you are changing value of them after request